docs/configuration/06-statusline.md

```diff
@@ -0,0 +1,132 @@
+# Statusline
+LunarVim uses `lualine` as a default statusline.
+
+Configuration is same as that of lualine with full support. See [Lualine README.md](https://github.com/shadmansaleh/lualine.nvim/blob/master/README.md)
+
+In addition, LunarVim provides predefined styles(layout) and components. 
+
+## Style
+There are three style options LunarVim accepts,
+- lvim   
+> LunarVim's default layout
+- default 
+> Lualine's default layout
+- none
+> Empty layout
+
+To set style other than `"lvim" style`,
+```lua
+lvim.builtin.lualine.style = "default" -- or "none"
```

Comment:
The style we're trying to maintain is to prefer code samples over explanatory text.  

To set the lualine style
```lua
  -- style = 'lvim' | 'default' | 'none'  
  lvim.builtin.lualine.style = 'lvim'
```
Then you can remove the explanation from above.  

```
There are three style options LunarVim accepts,
- lvim   
> LunarVim's default layout
- default 
> Lualine's default layout
- none
> Empty layout
```